python/jetToolbox_cff.py

```diff
@@ -26,7 +26,10 @@ def jetToolbox( proc, jetType, jetSequence, outputFile,
 		newPFCollection=False, nameNewPFCollection = '',	
 		PUMethod='CHS', 
 		miniAOD=True,
-		runOnMC=True,
+		runOnMC=True, 
+    jetRadius=0.4, ### To be specified for rerunning b tagging on subjets
+    algoLabel='', #### To be specified for rerunning b tagging on subjets 
+    fatJetSource='',  ### To be specified for rerunning b tagging on subjets
```

Comment:
@alefisico probably. but keep in mind that we still may want to independently set jetRadius and fatJetSource for the subjet b tagging part only.